protected/humhub/modules/admin/models/SpaceSearch.php

```diff
@@ -86,12 +86,16 @@ public function search($params)
         $dataProvider->setSort([
             'attributes' => [
                 'id',
+                'sort_order',
                 'name',
                 'visibility',
                 'join_policy',
                 'memberCount',
             ]
         ]);
+
+        $dataProvider->sort->defaultOrder = ['sort_order' => SORT_ASC];
```

Comment:
Would it perhaps be directly possible to add the `name` as a second argument here in addition to the `sort_order`?